master/internal/kubernetes/pod.go

```diff
@@ -48,7 +47,6 @@ type pod struct {
 	container        container.Container
 	ports            []int
 	resourcesDeleted bool
```

Comment:
question: If a trial stops and restarts, does it use the same pod actor?

Reply:
no we always use a new pod actor, since a trial restart is a new task